1_fetch/fetch_config.yaml

```diff
@@ -0,0 +1,68 @@
+sb_fetch.py:
+
+  metadata_id: 5e5c1c1ce4b01d50924f27e7
+  metadata_files:
+      - 1_fetch/out/metadata/lake_metadata.csv
+
+  obs_id: 5e5d0b68e4b01d50924f2b32
+  obs_files:
+    - 1_fetch/out/obs/temperature_observations.zip
+
+  driver_id: 5e5d0b96e4b01d50924f2b34
+  driver_files:
+    - 1_fetch/out/driver/04_inputs.xml
+    - 1_fetch/out/driver/clarity_01_N48.00-49.50_W89.50-97.25.zip
```

Comment:
If you use `expand` and separate out the "driver types" (`clarity`, `ice_flags`, `inputs`) you can let snakemake come up with all of the combinations of these.

* `expand` example: https://github.com/jsadler2/lake-temperature-lstm-static/blob/ef32613bf46a67df06a80195472ac4518718be28/Snakefile#L13

* breaking out "driver types" example: https://github.com/jsadler2/lake-temperature-lstm-static/blob/ef32613bf46a67df06a80195472ac4518718be28/1_fetch/fetch_config.yaml#L18

Reply:
I like how short the config file is with this change. However, rather than let Snakemake create the combinations, I moved that logic into the `get_input_files` function (now renamed as `get_mntoha_input_files`). That feels a bit easier to read and maintain to me?